src/class-wpml-cornerstone-translatable-nodes.php

```diff
@@ -143,6 +143,10 @@ private function get_wrap_tag( $settings ) {
 			return $settings['text_tag'];
 		}
 
+		if ( isset( $settings['_type'] ) && 'classic:custom-headline' === $settings['_type'] && isset( $settings['level'] ) ) {
+			return $settings['level'];
+		}
+
```

Comment:
@kagg-design 

How about reducing the duplication:
```
$seo_elements = array( 
    'headline' => 'text_tag',
    'classic:custom-headline' => 'level'
);

foreach( $seo_elements as $type => $attr ) {
    if ( isset( $settings['_type'] ) &&$type === $settings['_type'] && isset( $settings[ $attr ] ) ) {
			return $settings[ $attr ];
		}
}
```

Reply:
@brucepearson Good point, thanks, done.